src/content/docs/en/reference/experimental-flags/responsive-images.mdx

```diff
@@ -33,22 +41,37 @@ Additionally, Astro's image components can receive [responsive image props](#res
 
 Images in your `public/` folder are never optimized, and responsive images are not supported.
 
-## Responsive image configuration settings
+:::note
+Enabling responsive images will generate additional image sizes for all affected images. For prerendered pages this happens during the build so may increase the build time of your project, especially if you have a large number of images. For pages rendered on-demand the images are generated as-needed, so this has no impact on build times but may increase the number of transformations performed. Depending on your image service this may incur additional costs. 
```

Comment:
```suggestion
Enabling responsive images will generate additional image sizes for all affected images. For prerendered pages this happens during the build so may increase the build time of your project, especially if you have a large number of images.

For pages rendered on-demand the images are generated as-needed, so this has no impact on build times but may increase the number of transformations performed. Depending on your image service this may incur additional costs. 
```